hashin.py

```diff
@@ -128,8 +133,9 @@ def run_single_package(
     )
     package = data['package']
 
+    maybe_restriction = '' if restriction is None else '; {0}'.format(restriction)
```

Comment:
Because the None check, I can do:
```
$ hashin -r /tmp/reqs.txt "enum34==1.1.5  ; "
$ cat /tmp/reqs.txt
enum34==1.1.5;  \
    --hash=sha256:01a4abc326ead6c92169ab8c6f6af3cff9706a6a047c8738168f7a50b8def231 \
    --hash=sha256:35cf09a65db802269a76b7df60f548940575579a0e8a00ec45294995d28b0862 \
    --hash=sha256:7580699f440ebc08ce51e04efd34e685264779e64bb5312112a785eac769e037
```
Change it to `'' if not restriction else '; {0}'.format(restriction)` or `restriction and '; {0}'.format(restriction) or ''`